packages/eslint-plugin-pf-codemods/lib/helpers.js

```diff
@@ -278,26 +278,104 @@ function addCallbackParam(componentsArray, propMap) {
                     message: `The "${attribute.name.name}" prop for ${node.name.name} has been updated so that the "${newParam}" parameter is the first parameter. "${attribute.name.name}" handlers may require an update.`,
                     fix(fixer) {
                       const fixes = [];
-                      const createReplacerFix = (functionParam) => {
+
+                      const newParamHasUnderscore = newParam[0] === "_";
+
+                      const getIsNewParam = (param) => {
+                        if (param.name === newParam) {
+                          return true;
+                        }
+
+                        if (newParamHasUnderscore) {
+                          return param.name === newParam.slice(1);
+                        }
+
+                        return param.name === "_" + newParam;
+                      };
+
+                      // meant to preserve _ being used to signal that the param isn't being used (or not)
+                      const formatNewParam = (paramNameInCurrentUse = "_") => {
+                        const currentUseOfNewParamHasUnderscore =
+                          paramNameInCurrentUse[0] === "_";
+
+                        if (
+                          newParamHasUnderscore ===
+                          currentUseOfNewParamHasUnderscore
+                        ) {
+                          return newParam;
+                        }
+
+                        if (newParamHasUnderscore) {
+                          return newParam.slice(1);
+                        }
+
+                        return "_" + newParam;
+                      };
+
+                      const createParamAdditionFix = (
+                        params,
+                        paramNameInCurrentUse
+                      ) => {
+                        const firstParam = params[0];
+
+                        const replacementParams = `${formatNewParam(
+                          paramNameInCurrentUse
+                        )}, ${firstParam.name}`;
+
                         const hasParenthesis =
-                          context.getTokenAfter(functionParam).value === ")";
-                        const replacementParams = `${newParam}, ${functionParam.name}`;
+                          context.getTokenBefore(firstParam).value === "(";
+
+                        if (hasParenthesis) {
+                          return fixer.replaceText(
+                            firstParam,
+                            replacementParams
+                          );
+                        }
 
                         return fixer.replaceText(
-                          functionParam,
-                          hasParenthesis
-                            ? replacementParams
-                            : `(${replacementParams})`
+                          firstParam,
+                          `(${replacementParams})`
                         );
```

Comment:
Nit not a blocker: could keep the ternary for replaceText and just have one return; `(firstParam, hasParenthesis ? replacementText : (${replacementText}))`